src/services/NodeService.ts

```diff
@@ -111,19 +111,19 @@ export class NodeService {
     ): Promise<NodeModel> {
         try {
             const nodeInfo = await nodeGetter(paramValue);
-            if (!nodeInfo) {
-                return undefined;
+            if (nodeInfo) {
+                return this.createNodeModel(
+                    nodeInfo.apiStatus?.restGatewayUrl,
+                    networkType,
+                    nodeInfo.friendlyName,
+                    true,
+                    nodeInfo.publicKey,
+                    nodeInfo.apiStatus?.nodePublicKey,
+                    nodeInfo.apiStatus?.webSocket?.url,
+                );
             }
-            return this.createNodeModel(
-                nodeInfo.apiStatus?.restGatewayUrl,
-                networkType,
-                nodeInfo.friendlyName,
-                true,
-                nodeInfo.publicKey,
-                nodeInfo.apiStatus?.nodePublicKey,
-                nodeInfo.apiStatus?.webSocket?.url,
-            );
         } catch (error) {
+            console.log(error);
```

Comment:
no need to send to the form?

Reply:
Thanks for the comment mate, what do you mean by _form_?

Reply:
I mean if it errors out, should the frontend show sth in the notification popup?